etn-ccis/blui-react-native-workflows
# Feature/5511 error manager to support string interpolation

Fixes # BLUI-5511



#### Changes proposed in this Pull Request:

- Updated error manager to support string interpolation
- Updated docs
- Updated example



#### Screenshots / Screen Recording (if applicable)

- 
![Simulator Screenshot - iPhone 12 - 2024-03-28 at 17 35 03](https://github.com/etn-ccis/blui-react-native-workflows/assets/120575281/16f2862b-0eb5-4280-9ade-30b17c9d6e91)




#### To Test:

- cd login-workflows
- Press Registration Provider Example
- Go through registration process
- You will see error 



#### Any specific feedback you are looking for?

- **Note: I will remove the test code added for error**
